src/main/webapp/script.js

```diff
@@ -226,8 +253,8 @@ function fetchMentorApproval() {
       const paragraphElement = document.getElementById('paragraph');
       paragraphElement.innerHTML = approval.paragraph;
     } else {
-      // If approver is not assigned to mentor, redirect to index.
-      window.location.replace('/index.html');
+      // If user is not either a mentor or an approver assigned to that mentor, redirect to index.
+      //window.location.replace('/index.html');
```

Comment:
why is this line commented? what happens if it actually goes to the else statement?

Reply:
Was doing some testing, forgot to uncomment.